packages/react-components/react-rating-preview/src/components/Rating/useRating.tsx

```diff
@@ -27,16 +27,12 @@ export const useRating_unstable = (props: RatingProps, ref: React.Ref<HTMLDivEle
     iconFilled = <StarFilled />,
     iconOutline = <StarRegular />,
     max = 5,
-    mode = 'interactive',
     name = generatedName,
     onChange,
     step = 1,
     size = 'medium',
```

Comment:
The default size for the interactive version is `extra-large` according to the design.
```suggestion
    size = 'extra-large',
```

Also update the `@default` comment in Rating.types.ts.